### PR TITLE
Feature: database structure

### PR DESCRIPTION
**Pull request by filenkoB**

### CODE REVIEW

File: backend/src/Domain/Entities/User.cs [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
         }
+        public string Password { get; set; }
+        public string PasswordSalt { get; set; }
 
-        public string FirstName { get; set; }
-        public string LastName { get; set; }
-        public DateTime Birth { get; set; }
-        public UserStatus Status { get; set; }
-
+        public Vacancy Vacancy { get; private set; }
```
Comment (anton-matsyshyn) [inline backend/src/Domain/Entities/User.cs:17]: Why user (hr) has only _one_ vacancy and refresh token? Better to have list here

File: backend/src/Domain/Entities/User.cs [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+        public RefreshToken RefreshToken { get; private set; }
+        public ICollection<UserToRole> UserRoles { get; private set; }
```
Comment (anton-matsyshyn) [inline backend/src/Domain/Entities/User.cs:19]: I suppose EF core cannot access private setters to load related data, make them public (as default)

File: backend/src/Domain/Entities/Applicant.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using Domain.Entities.Abstractions;
+
+namespace Domain.Entities
+{
+    public class Applicant : Human
+    {
+        public string Phone { get; set; }
+        public string Skype { get; set; }
+        public string Experience { get; set; }
```
Comment (anton-matsyshyn) [inline backend/src/Domain/Entities/Applicant.cs:11]: I think double fits better here (same for Candidate)

**Comment by filenkoB** [comment]
Fixed

**Review by anton-matsyshyn — APPROVED** [review]

**Pull request merged**